pytest-embedded-idf/tests/test_idf.py

```diff
@@ -348,6 +348,46 @@ def test_detect_port(dut):
     result.assert_outcomes(passed=1)
 
 
+def test_check_panic_output(testdir):
+    testdir.makepyfile(r"""
+        import pexpect
+        import pytest
+
+        def test_check_panic_output(dut):
+            dut.expect('Stack memory')
+            dut.expect('register dump')
+    """)
+
+    result = testdir.runpytest(
+        '-s',
+        '--embedded-services', 'esp,idf',
+        '--app-path', f'{os.path.join(testdir.tmpdir, "hello_world_esp32c3_panic")}',
+        '--target', 'esp32c3',
+    )
+
+    result.assert_outcomes(passed=1)
+
+
+def test_skip_check_panic_output(testdir):
+    testdir.makepyfile(r"""
+        import pexpect
+        import pytest
+
+        def test_check_panic_output(dut):
+            assert dut.expect('register dump')
```

Comment:
assert not here?

this is the same statement as the `test_check_panic_output` one `dut.expect('register dump')`. These two statement should be different, right?